File: src/pages/exercises/basic-irrf/_codes/php/response/irrf.php
<?php

const salary = 3500;
$tax;

switch (true) {
  case salary < 1903.99:
    $tax = 0;
    break;
  case salary < 2826.66:
    $tax = 0.075 * salary - 142.8;
    break;
  case salary < 3751.06:
    $tax = 0.15 * salary - 354.8;
    break;
  case salary < 4664.68:
    $tax = 0.225 * salary - 636.13;
    break;
  default:
    $tax = 0.275 * salary - 869.36;
}

print("IRRF: ${tax}");

File: problems/algorithms/basic-irrf/ruby/response/irrf.rb
salary = 3500
tax = 0

if salary < 1903.99
  tax = 0
elsif salary < 2826.66
  tax = 0.075 * salary - 142.8
elsif salary < 3751.06
  tax = 0.15 * salary - 354.8
elsif salary < 4664.68
  tax = 0.225 * salary - 636.13
else
  tax = 0.275 * salary - 869.36
end

puts "IRRF: #{tax}"
